internal/ld/ld.go

```diff
@@ -389,11 +392,59 @@ func (b BranchRep) TotalHunkCount() int {
 	return count
 }
 
+func (b BranchRep) WriteToCSV(outDir, projKey, repo, sha string) (path string, err error) {
+	tag := b.Name
+	if len(sha) >= 7 {
+		tag = sha[:7]
+	}
+
+	absPath, err := validation.NormalizeAndValidatePath(outDir)
+	if err != nil {
+		return "", fmt.Errorf("invalid outDir '%s': %s", outDir, err)
+	}
+	path = filepath.Join(absPath, fmt.Sprintf("coderefs_%s_%s_%s.csv", projKey, repo, tag))
+
+	f, err := os.Create(path)
```

Comment:
Will this create the parent directory if it doesn't already exist?

Reply:
Nevermind; now I came across https://github.com/launchdarkly/ld-find-code-refs/pull/97/files#diff-bd40f10a5bef7a0945987430ac7762bdR21

Reply:
Good find-- yes, the validation code is called at the start of execution in `options.go`, ensuring that the directory does not exist. I'll add a note in the readme that the directory must already exist.